resources/deb/etc/init.d/openhab2

```diff
@@ -71,15 +71,36 @@ fi
 # Load the VERBOSE setting and other rcS variables
 . /lib/init/vars.sh
 
+# Check to see if openHAB is currently running
+EXISTINGPID=`pgrep java -G openhab | tail -1`
```

Comment:
If it is then the variable will contain the latest process that the _openhab_ group has opened. I think this is fine because processes aren't killed based on the contents on the file and it is merely an indicator of that openHAB is doing something.